ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/ContractsReader.scala

```diff
@@ -55,6 +63,45 @@ private[dao] sealed abstract class ContractsReader(
           )
       })
 
+  /** Lookup of a contract in the case the contract value is already known (loaded from a cache) */
+  private[this] def lookupActiveContractWithoutArgument(
```

Comment:
```suggestion
  private[this] def lookupActiveContractWithoutArgument(
```
1. if you limit the visibility to the instance to make the runtime works easier when it comes to inlining, consider that `private` is enough for methods (`private[this]` has an effect on `val`s)
2. are you sure the name is correct? Probably `lookupCachedActiveContract` would be less ambiguous.